assets/src/stories-editor/components/font-family-picker/status.js

```diff
@@ -0,0 +1,90 @@
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+
+export default class Status extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			bump: false,
+		};
+	}
+
+	componentWillReceiveProps( { queryLength } ) {
```

Comment:
This is [a deprecated method](https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops). We should most definitely try to avoid it.

Why is it necessary?

Reply:
Looks like it was just copied over from upstream: https://github.com/alphagov/accessible-autocomplete/blob/baee8582ae28ec499289eded99053e0c27ebf33d/src/status.js#L22-L27

Reply:
Yeah, I figured out the reasoning (a11y), however it's a bit flawed as the aria live component only works if on screen at first HTML render - which it obviously won't be. So I don't think it works anyway.

It did rewrite the component in 37a3355 to avoid these silly work-arounds.